proto/v1/event.proto

```diff
@@ -70,5 +70,21 @@ message EventCollectionRequest {
 }
 
 message EventCollectionResponse { 
-    EventCollection event_collection = 1;
+  EventCollection event_collection = 1;
 }
+
+message EventCollectionDate {
+  // Format: YYYY-MM, corresponding to the date in EventCollection cache key.
+  // Sorted from earliest to latest.
+  repeated string dates = 1;
+}
+
+message EventCollectionDateRequest {
+  string event_type = 1;
+  // Only for top-level places: Earth, continent, country, state, adminArea1.
+  string affected_place_dcid = 2;
+}
+
+message EventCollectionDateResponse {
+  EventCollectionDate event_collection_date = 1;
```

Comment:
curious why we need the EventCollectionDate type when it's just a single `repeated string`?

Reply:
For one thing, we need the EventCollectionDate to do type casting for the data read from BT, see event.go. 
Regarding here, I guess either is fine, but the intention is to make the API a transparent proxy to the BT cache.